Managers/UInputMan.h

```diff
@@ -1,1684 +1,815 @@
+
+		/// <summary>
+		/// Shows whether a mouse button was pressed between the last update and the one previous to it.
+		/// </summary>
+		/// <param name="whichButton">Which button to check for.</param>
+		/// <param name="whichPlayer">Which player to check for.</param>
+		/// <returns>Pressed or not.</returns>
+		bool MouseButtonPressed(int whichButton, short whichPlayer);
+
+		/// <summary>
+		/// (ONLY FOR LUA BACKWARD COMPATIBILITY) Shows whether a mouse button was pressed between the last update and the one previous to it.
+		/// </summary>
+		/// <param name="whichButton">Which button to check for.</param>
+		/// <returns>Pressed or not.</returns>
+		bool MouseButtonPressed(int whichButton) { return MouseButtonPressed(whichButton, 0); };
+
+		/// <summary>
+		/// Shows whether a mouse button was released between the last update and the one previous to it.
+		/// </summary>
+		/// <param name="whichButton">Which button to check for.</param>
+		/// <param name="whichPlayer">Which player to check for.</param>
+		/// <returns>Released or not.</returns>
+		bool MouseButtonReleased(int whichButton, short whichPlayer);
+
+		/// <summary>
+		/// (ONLY FOR LUA BACKWARD COMPATIBILITY) Shows whether a mouse button was released between the last update and the one previous to it.
+		/// </summary>
+		/// <param name="whichButton">Which button to check for.</param>
+		/// <returns>Released or not.</returns>
+		bool MouseButtonReleased(int whichButton) { return MouseButtonReleased(whichButton, 0); };
+
+		/// <summary>
+		/// Shows whether the mouse wheel has been moved past the threshold limit in either direction this frame.
+		/// </summary>
+		/// <returns>The direction the mouse wheel has been moved which is past that threshold. 0 means not past, negative means moved down, positive means moved up.</returns>
+		int MouseWheelMoved() const { return m_MouseWheelChange; }
+
+		/// <summary>
+		/// Gets the relative mouse wheel position for the specified player.
+		/// </summary>
+		/// <param name="player">The player to get mouse wheel position for.</param>
+		/// <returns>The relative mouse wheel position for the specified player.</returns>
+		int MouseWheelMovedByPlayer(short player) const { return (m_OverrideInput && player >= 0 && player < MAX_PLAYERS) ? m_NetworkMouseWheelState[player] : m_MouseWheelChange; }
+
+		/// <summary>
+		/// Return true if there is any mouse button presses at all.
+		/// </summary>
+		/// <returns>Whether any mouse buttons have been pressed at all since last frame.</returns>
+		bool AnyMouseButtonPress();
+
+		/// <summary>
+		/// Sets the mouse to be trapped in the middle of the screen so it doesn't go out and click on other windows etc.
+		/// This is usually used when the cursor is invisible and only relative mouse movements are used.
+		/// </summary>
+		/// <param name="trap">Whether to trap the mouse or not.</param>
+		/// <param name="whichPlayer">
```

Comment:
This one is too long.

Reply:
Should have been more clear, I don't mean the super short ones, I mean the longer ones - `SetMousePos` and `ForceMouseWithinBox` both have `Which player is trying to control the mouse. Only the player with actual control over the mouse will be affected. -1 means do it regardless of player.` on oneline while this has pretty much the same thing on two.

Reply:
This one is too long.
The vile horizontal devil scrollbar is already summoned because long lines but this one will force me to use it. No thank.
Pls send 3840×1080 display.

Reply:
Hahaha alright, whatever. If you don't wanna oneline this that's fair, though then I'd probably rather they all be 2-line so they look more consistent, but it's not a big deal either way.